src/utils/surfaces.py

```diff
@@ -4,56 +4,79 @@
 
 from src.enums import ElementSubType, ElementType
 from src.utils.assets import (
+    BIG_BUSH,
+    MEDIUM_BUSH,
+    SMALL_BUSH,
     BIG_CLOUD,
     MEDIUM_CLOUD,
     SMALL_CLOUD,
+    BIG_MOUNTAIN,
+    MEDIUM_MOUNTAIN,
+    SMALL_MOUNTAIN,
+    BIG_TREE,
+    MEDIUM_TREE,
+    SMALL_TREE,
     OVERWORLD_BLOCK,
-    OVERWORLD_BRICK,
-    OVERWORLD_BROKE_BRICK,
-    MASTERY_BLOCK_1,
-    MASTERY_BLOCK_2,
-    PIPE_BODY_STYLE_1,
-    PIPE_BODY_STYLE_2,
-    PIPE_BODY_STYLE_3,
-    PIPE_BODY_STYLE_4,
+    WALL_BLOCK,
+    MYSTERY_BLOCK_1,
+    MYSTERY_BLOCK_2,
     PIPE_HEAD,
+    PIPE_BODY,
```

Comment:
The idea is that the pipes have different styles to be able to give it different aspects and not be so equal.

Reply:
It will be fixed, before uploading the feature